lib/mix/lib/mix/tasks/test.ex

```diff
@@ -168,13 +170,28 @@ defmodule Mix.Tasks.Test do
   been changed since the last run with `--stale`.
   """
 
-  @switches [force: :boolean, color: :boolean, cover: :boolean,
-             trace: :boolean, max_cases: :integer, include: :keep,
-             exclude: :keep, seed: :integer, only: :keep, compile: :boolean,
-             start: :boolean, timeout: :integer, raise: :boolean,
-             deps_check: :boolean, archives_check: :boolean, elixir_version_check: :boolean,
-             stale: :boolean, listen_on_stdin: :boolean, formatter: :keep,
-             slowest: :integer]
+  @switches [
+    force:                :boolean,
+    color:                :boolean,
+    cover:                :boolean,
+    trace:                :boolean,
+    max_cases:            :integer,
+    include:              :keep,
+    exclude:              :keep,
+    seed:                 :integer,
+    only:                 :keep,
+    compile:              :boolean,
+    start:                :boolean,
+    timeout:              :integer,
+    raise:                :boolean,
+    deps_check:           :boolean,
+    archives_check:       :boolean,
+    elixir_version_check: :boolean,
+    stale:                :boolean,
+    listen_on_stdin:      :boolean,
+    formatter:            :keep,
+    slowest:              :integer
+  ]
```

Comment:
Hey @jstumbaugh, we're not allowing any stylistic chances after the formatter is run because otherwise the next time the formatter is run on this file it will keep changing it back (in this case, back to not aligned).